simplified-app-shared/src/main/java/org/nypl/simplified/app/catalog/CatalogFeedActivity.java

```diff
@@ -748,14 +733,32 @@ public Boolean onFeedSearchLocal(
             return NullCheck.notNull(Boolean.TRUE);
           }
         });
+
+    } else {
+      CatalogFeedActivity.LOG.debug("Feed has no search opts.");
     }
 
     if (search_ok) {
+      this.search_view.setSubmitButtonEnabled(true);
+      this.search_view.setIconifiedByDefault(false);
+      search_item.setShowAsActionFlags(MenuItem.SHOW_AS_ACTION_ALWAYS);
+      search_item.expandActionView();
+
+      this.search_view.setQueryHint("Search " + this.feed.getFeedTitle());
```

Comment:
I believe this string should be localized. We should also avoid using string concatenation here (and use a format string instead) so that different languages can handle things appropriately.

Reply:
sure